torch/distributed/rpc/__init__.py

```diff
@@ -159,13 +156,12 @@ def init_rpc(
             backend = BackendType.TENSORPIPE  # type: ignore[attr-defined]
 
         if backend == BackendType.PROCESS_GROUP:  # type: ignore[attr-defined]
```

Comment:
Do we also plan to remove `PROCESS_GROUP` from `BackendType`?

Reply:
We can remove it and the user will get an `AttributeError` if they have existing code that uses it. Or we can keep the enum type (remove rest of code) for a release and throw a runtime error with description when the user tries to use it. Is it better to just remove it altogether?

Reply:
Make sense. Keeping the enum item for one more release might be more user-friendly.